hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/shell/TestOzoneRepairShell.java

```diff
@@ -80,20 +87,23 @@ static void cleanup() {
     IOUtils.closeQuietly(cluster);
   }
 
-  @Test
-  public void testUpdateTransactionInfoTable() throws Exception {
+  @ParameterizedTest
+  @EnumSource(value = Component.class, names = {"OM", "SCM"})
+  public void testUpdateTransactionInfoTable(Component component) throws Exception {
     CommandLine cmd = new OzoneRepair().getCmd();
-    String dbPath = new File(OMStorage.getOmDbDir(conf) + "/" + OM_DB_NAME).getPath();
+    String dbPath = getDbPath(component);
+    String componentLowerCase = component.toString().toLowerCase();
```

Comment:
Prefer `Enum#name()`, `toString()` may be overridden.

```suggestion
    String componentLowerCase = component.name().toLowerCase();
```